packages/mui-base/src/useBadge/index.ts

```diff
@@ -0,0 +1,3 @@
+export { default as useBadge } from './useBadge';
+export * from './useBadge';
+export { default } from './useBadge';
```

Comment:
We can drop the named export, so it's consistent with other components:

```suggestion
export { default } from './useBadge';
export * from './useBadge';
```

(here and in other files)